Musakai.Shared/Api/Contact/Requests/ContactRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Musakai.Shared.Api.Contact.Requests;
+
+/// <summary>
+/// Request for sending a contact message to the Photographer
+/// </summary>
+public class ContactRequest
+{
+    /// <summary>
+    /// Name of the person sending the message
+    /// </summary>
+    [Required(ErrorMessage = "{0} is required.")]
+    [Display(Name = "First Name")]
+    public string Name { get; set; } = string.Empty;
```

Comment:
Why there is an string.Empty when based on line 13 it should be required? 🤔

Reply:
This is because we ensure that the value is never null. Required  attribute also checks, if the length of the string is more then 0. This is mainly for compiler, that will have information, that there will never come nullable values even if there will not be any value. In the end, both, ModesState and Blazor will still validate the form based on the attribute, which means there will be errors while trying to send the Empty string. API will deny this request and Blazor should Validate this and display error message.